dlib/dnn/loss.h

```diff
@@ -1397,12 +1397,34 @@ namespace dlib
 
             out << "detector_windows:(";
             auto& opts = item.options;
-            for (size_t i = 0; i < opts.detector_windows.size(); ++i)
+
             {
```

Comment:
Remove { if removing the `for` loop

Reply:
Actually wanted to have that new stuff in a scope of its own. But extracted this now to an `operator <<`.